infra/aws/terraform/management-account/organization-accounts-boskos.tf

```diff
@@ -41,3 +40,167 @@ module "k8s_infra_e2e_boskos_003" {
   iam_user_access_to_billing = "ALLOW"
   parent_id                  = aws_organizations_organizational_unit.boskos.id
 }
+
+##############################################
+# Accounts used by boskos in the EKS cluster #
+##############################################
+
+module "k8s_infra_e2e_boskos_004" {
+  source = "../modules/org-account"
+
+  account_name               = "k8s-infra-e2e-boskos-004"
+  email                      = "k8s-infra-aws-admins+k8s-infra-e2e-boskos-004@kubernetes.io"
+  iam_user_access_to_billing = "ALLOW"
+  parent_id                  = aws_organizations_organizational_unit.boskos.id
+
+  tags = {
+    "production"  = "true",
+    "environment" = "prod",
+    "group"       = "sig-k8s-infra",
+    "service"     = "boskos"
+  }
```

Comment:
I've seen that we add tags for all new AWS accounts, so I added some tags. I'm not sure if those tags are correct and needed, so please check.

Reply:
We currently only have policies for keys `service` and `group`. The rest of the tags are for the moment optional.

Reply:
@ameukam Should I change something or leave as it is?

Reply:
You can leave this as it is. Tags are mutable and can be updated later.